project/plugins.sbt

```diff
@@ -1,9 +1,8 @@
 addSbtPlugin("com.geirsson"              % "sbt-ci-release"    % "1.5.3")
-addSbtPlugin("org.scalameta"             % "sbt-mdoc"          % "2.1.5")
-addSbtPlugin("org.scalameta"             % "sbt-scalafmt"      % "2.3.4")
+addSbtPlugin("org.scalameta"             % "sbt-mdoc"          % "2.2.0")
```

Comment:
weird that scala-steward didn't care for those

Reply:
I think Scala Steward isn't working as it should... hopefully that will change with the next version of `.github`... 😈